lots of text adjustments, flavor text additions, leader ceremonies

yippeecore opened the pull request:
slight text adjustments to various files, i also added onto some of the failed outcomes for the abandoned kittypet & injured cat patrols to try and expand why outsider rep worsens. ~~wrote 3 new nine life ceremonies for strange cats as well.~~ took em out due to feature freeze!

yippeecore committed:
lots of text adjustments, flavor text additions, leader ceremonies

slight text adjustments to various files, i also added onto some of the failed outcomes for the abandoned kittypet & injured cat patrols to try and expand why outsider rep worsens. wrote 3 new nine life ceremonies for strange cats as well.

yippeecore committed:
otherclan_hostile2 typo fixes

removed 'the' from the intro text and fixed 2 spacing issues i noticed

yippeecore committed:
Update lead_ceremony_sc.json

removed the new strange_warrior flavor texts i wrote due to the current feature freeze

yippeecore committed:
corrected a small spacing issue

spikybell to spiky bell

yippeecore committed:
uncapitalized warrior code

yippeecore committed:
capitalization fix

yippeecore committed:
misc text fixes

yippeecore committed:
Merge remote-tracking branch 'upstream/development' into development

Thlumyn commented on resources/dicts/events/misc_events/general/general.json:
I think this is supposed to be capitalized

Thlumyn commented on resources/dicts/patrols/general/training.json:
```diff
@@ -9386,8 +9386,8 @@
-        "intro_text": "It is almost dawn when the deputy rouses app1 for {PRONOUN/app1/poss} assessment. {PRONOUN/app1/subject/CAP} are meant to find p_l posing as an enemy warrior somewhere on the territory and handle the situation. p_l could be anywhere!",
-        "decline_text": "app1 sheepishly looks up at the deputy, a guilty gleam in {PRONOUN/app1/poss} eyes, and says that {PRONOUN/app1/subject} have a stomachache. They {PRONOUN/app1/subject/CAP} hurriedly {VERB/app1/run/runs} to the medicine den, wondering how {PRONOUN/app1/subject} are going to get away with this fake illness at sunhigh when the Clan shares tongues.",
+        "intro_text": "It's almost dawn when the deputy rouses app1 for {PRONOUN/app1/poss} assessment. {PRONOUN/app1/subject/CAP} are meant to find p_l posing as an enemy warrior somewhere on the territory and handle the situation. p_l could be anywhere!",
+        "decline_text": "app1 sheepishly looks up at the deputy, a guilty gleam in {PRONOUN/app1/poss} eyes, and says that {PRONOUN/app1/subject} have a stomachache. {PRONOUN/app1/subject/CAP} hurriedly {VERB/app1/run/runs} to the medicine den, wondering how {PRONOUN/app1/subject}'re going to get away with this fake illness at sunhigh when the Clan shares tongues.",
```
{PRONOUN/app1/subject}'re 
Looks like it has some pronoun tagging issues, might need a 's/'re otherwise you'd end up with things like "wondering how he're going to get away"

Thlumyn commented on resources/dicts/thoughts/alive/general.json:
```diff
@@ -1061,8 +1061,8 @@
             "Brags about all the prey {PRONOUN/m_c/subject} caught when {PRONOUN/m_c/subject} {VERB/m_c/were/was} a rogue",
-            "Can't help but be intrigued by the Warrior Code",
-            "Is trying to memorize the Warrior Code"
+            "Can't help but be intrigued by the warrior code",
```
I think this part is supposed to be capitalized

yippeecore committed:
Update training.json

fixed a pronoun tagging issue brought up here - https://github.com/ClanGenOfficial/clangen/pull/2116#discussion_r1534328731

yippeecore committed:
Update general.json

fixed a capitalization issue brought up here - https://github.com/ClanGenOfficial/clangen/pull/2116#discussion_r1534324519

yippeecore committed:
Update general.json

fixed some capitalization concerns brought up here - https://github.com/ClanGenOfficial/clangen/pull/2116#discussion_r1534355387

Lixxis commented on resources/dicts/events/nutrition/leader.json:
```diff
@@ -8,7 +8,7 @@
             "death"
         ],
-        "death_text": "m_c feels that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have enough energy to stand up and remain here.",
+        "death_text": "m_c feels that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have enough energy to stand up and remain here as starvation claws into {PRONOUN/m_c/poss} belly and sends {PRONOUN/m_c/object} to {PRONOUN/m_c/poss} ancestral hunting grounds.",
```
Be careful here, you add this line to the leader, but the leader might still have multiple life's left. @Tiritirimatangi you have the lead in the writing part.

Tiritirimatangi replied:
I think we should delete the "and sends {PRONOUN/m_c/object} to {PRONOUN/m_c/poss} ancestral hunting grounds". Hopefully soon an event rewrite will allow us to separate regular deaths and leader deaths like the patrols can do!

Thlumyn replied:
@yippeecore

Thlumyn commented:
Closing as stale. Changes requested >1 month ago. 

Please resubmit if you still plan on working on this; though you might want to split it up into multiple PRs.
